<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
  * Released as open source by Gabriel Caudrelier
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
  *
  * https://github.com/metrodango/pip3line
  *
  * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>SplitWidget</class>
 <widget class="QWidget" name="SplitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="separatorLabel">
     <property name="text">
      <string>Separator</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="groupLabel">
     <property name="text">
      <string>Target group</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="groupSpinBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="allCheckBox">
     <property name="text">
      <string>Just take all groups (one per line)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="trimCheckBox">
     <property name="text">
      <string>Remove any blank characters</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="procLineBLinecheckBox">
     <property name="toolTip">
      <string>Process input line by line as opposed to one block.</string>
     </property>
     <property name="text">
      <string>Process input line by line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="HexWidget" name="charWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexWidget</class>
   <extends>QWidget</extends>
   <header location="global">hexwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
